GA-worksheet.md

```diff
@@ -7,62 +7,62 @@
 
 ##Project #2 Part #1: Header HTML
 1. What is the purpose of the header tag?
-
+The purpose of a header tag is for headers.
```

Comment:
Oh, okay. Thank you. Should I fix what I said then?
On Oct 25, 2015 10:54 AM, "Denny Tek" <notifications@github.com> wrote:

> In GA-worksheet.md
> <https://github.com/tekd/ga-worksheet/pull/5#discussion_r42945690>:
>
> > @@ -7,62 +7,62 @@
> >
> >  ##Project #2 Part #1: Header HTML
> >  1. What is the purpose of the header tag?
> > -
> > +The purpose of a header tag is for headers.
>
> This is correct, but a little vague! The header tag actually holds the
> introductory material for the website, such as a banner, logo, or
> navigational links. Check it out on Buzzfeed.com. The top of the page that
> lets you know that you're on Buzzfeed is the header (inspect element to
> clarify). It sits at the top of the body.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tekd/ga-worksheet/pull/5/files#r42945690>.
>